src/checks/__test__/validateJson.test.ts

```diff
@@ -0,0 +1,160 @@
+import { decodeJson, parseJson, readFileSync } from "../validateJson";
+import { right } from "fp-ts/lib/Either";
+import { VersionInfo } from "../../../generated/definitions/content/VersionInfo";
+import { Zendesk } from "../../../generated/definitions/content/Zendesk";
+
+const expectedValidJson =
+  "{\n" +
+  '  "min_app_version": {\n' +
+  '    "ios": "1.27.0",\n' +
+  '    "android": "1.27.0"\n' +
+  "  },\n" +
+  '  "min_app_version_pagopa": {\n' +
+  '    "ios": "0.0.0",\n' +
+  '    "android": "0.0.0"\n' +
+  "  },\n" +
+  '  "latest_released_app_version" : {\n' +
+  '    "ios": "2.1.0.1",\n' +
+  '    "android": "2.1.0.1"\n' +
+  "  },\n" +
+  '  "rollout_app_version" : {\n' +
+  '    "ios": "0.0.0",\n' +
+  '    "android": "0.0.0"\n' +
+  "  }\n" +
+  "}";
+
+const expectedValidJsonObject = {
+  min_app_version: {
+    ios: "1.27.0",
+    // tslint:disable-next-line:object-literal-sort-keys
```

Comment:
If we put this line at the very start of the file
`/* tslint:disable:object-literal-sort-keys */`
we can avoid to repeat it in every occurrence

Reply:
TSLint references removed in cd0ebb15da23819fd62a4c8dfcd7eb3dc8f30f07